contracts/adapters/swap/injective/src/contract.rs

```diff
@@ -48,6 +48,12 @@ pub enum InjectiveQueryMsg {
     },
 }
 
+#[cw_serde]
+pub struct FPCoin {
+    pub amount: Uint128,
```

Comment:
`Uint128` is not the right type here. I have deployed a version that uses this and it will panic if the value has a decimal https://github.com/InjectiveLabs/swap-contract/blob/master/contracts/swap/src/types.rs#L14